src/first.h

```diff
@@ -55,4 +55,8 @@
 
 #define UNUSED(x) ( (void)(x) )
 
+#ifdef HAVE_WOLFSSL_SSL_H
```

Comment:
This does not belong here.  There is only compiler and C library setup here.  What was your reasoning for adding something so wolfSSL specific here, to be included in every file?

If the other modules which check for "#if defined HAVE_LIBSSL && defined HAVE_OPENSSL_SSL_H" should be isolated into a new lighttpd header, such as sys-crypto.h, then let's create such an abstraction.